addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -209,21 +209,11 @@
 				<aligny>center</aligny>
 				<visible>VideoPlayer.HasEpg</visible>
 			</control>
-			<control type="label">
-				<left>260</left>
-				<right>50</right>
-				<top>20</top>
-				<height>25</height>
-				<label>$INFO[VideoPlayer.Title]$INFO[VideoPlayer.EpisodeName, (,)]</label>
-				<align>left</align>
-				<font>font36_title</font>
-				<aligny>center</aligny>
-			</control>
```

Comment:
I added episodename to topbar with this PR and show title was displayed in top bar and seek her. Take a look at the screen shots, please.